src/view/components/rule-engine/condition-types/serverside-condition/serverside-condition.ts

```diff
@@ -86,10 +102,24 @@ export class ServersideCondition {
     this._inputs = [];
   }
 
+  setVisible(value, input):void {
+    if(!value) return
+    if(input.name === 'comparison'){
+      let idx = this._inputs.indexOf(input)
+      let comparisonObj = input.options.filter((e)=> { return e.value == value })[0]
```

Comment:
SESE. When nesting seems crazy, that means your method is doing too much on its own - create more support methods.